src/main/scala/io/archivesunleashed/matchbox/ExtractBoilerpipeText.scala

```diff
@@ -29,29 +29,13 @@ object ExtractBoilerpipeText {
    * @return text with boilerplate removed or Nil if the text is empty.
    */
   def apply(input: String): String = {
-    try {
-      if (input.isEmpty) {
-        null
-      } else {
-        extract(input)
-      }
-    } catch {
-      case e: Exception =>
-        throw new IOException("Caught exception processing input row " + e)
-    }
-  }
-
-  /** Extracts boilerplate.
```

Comment:
Let's keep `ExtractBoilerpipeText` - it's one of our core functions, is documented, and is out of scope for this PR in any case (I suspect its removal is accidental?).

Reply:
I believe the code on the right does the exact same job as the code on the left. I do not think the exception is needed. If it is, I think it would be better to catch the more specific issue coming from Exception and handle it accordingly with more detailed warnings. But I have not been able to think of a case, barring something really crazy hitting the server.

Reply:
Yeah - sorry.  In the view I was using the changed code was on the right and the old code was on the left.

Reply:
More clearly, the `extract` method is incorporated into `apply` with no (known) differences in the way ExtractBoilerplateText works.

Reply:
Tested - and it works.